src/bin/vip-import-check.js

```diff
@@ -0,0 +1,180 @@
+#!/usr/bin/env node
+// @flow
+
+/**
+ * External dependencies
+ */
+const readline = require( 'readline' );
+const fs = require( 'fs' );
+import chalk from 'chalk';
```

Comment:
We should also standardize these to `import..from` as well

Reply:
+1

Reply:
Added in https://github.com/Automattic/vip/pull/566/commits/8f50d540602a7c974cf73d5eaec6f20bdfc32b7f